docs/guides/launch.md

```diff
@@ -54,290 +54,93 @@ access it at `http://127.0.0.1:3000/` under credentials `admin/admin`.
 > If you don't see any data displayed on the Grafana dashboards - try setting the timeframe to "Last 30 minutes". You
 > will also have to have `jq` installed on your system.
 
-## (Re)deploy db and contracts
-
-```
-zk contract redeploy
-```
-
-## Environment configurations
+## Ecosystem Configuration
 
-Env config files are held in `etc/env/target/`
+Ecosystem configuration can be found in:
 
-List configurations:
+- `/ZkStack.yaml`
+- `/configs` folder
+- `/chains` folder
 
-```
-zk env
-```
-
-Switch between configurations:
-
-```
-zk env <ENV_NAME>
-```
-
-Default configuration is `dev.env`, which is generated automatically from `dev.env.example` during `zk init` command
-execution.
+These files are created at ecosystem initialization `zkstack ecosystem init` and at chain initialization
+`zkstack chain init`.
 
 ## Build and run server
 
 Run server:
 
+```bash
+zkstack server
 ```
-zk server
-```
-
-Server is configured using env files in `./etc/env` directory. After the first initialization, file
-`./etc/env/target/dev.env`will be created. By default, this file is copied from the `./etc/env/target/dev.env.example`
-template.
 
-Make sure you have environment variables set right, you can check it by running: `zk env`. You should see `* dev` in
-output.
+The server's configuration files can be found in `/chains/<chain_name>/configs` directory. These files are created when
+running `zkstack chain init` command.
```

Comment:
How do you modify the config? Should you edit those files directly?

Reply:
Added a section on this matter in https://github.com/matter-labs/zksync-era/pull/3134/commits/1956a2e21c61e3801c6348bac9dbd75897c53eb5 @Deniallugo is this aligned with our general recommendations?